lib/inv_ca_cg.cpp

```diff
@@ -1,5 +1,9 @@
 #include <invert_quda.h>
 #include <blas_quda.h>
+#ifdef HAVE_OPENBLAS
```

Comment:
Relevant for multiple files, might be a good idea to have a `eigen_helper.h` (or whatever) that includes the `#ifdef HAVE_OPENBLAS`, #include<Eigen/Dense>`, etc in one place. It shouldn't hurt to have an overcomplete set of `Eigen` headers (`Dense`, `Eigenvalues`, etc) included in a given `eigen_helper.h` file.